app/uk/gov/hmrc/cgtpropertydisposals/models/upscan/UpscanRequest.scala

```diff
@@ -18,8 +18,11 @@ package uk.gov.hmrc.cgtpropertydisposals.models.upscan
 
 import play.api.libs.json.Json
 
-final case class UpscanInitiateReference(value: String) extends AnyVal
+final case class UploadRequest(
```

Comment:
the file name needs updating here

Reply:
This model is to absorb the response coming back from upscan-initiate.

Reply:
The filename will be in the fields map

Reply:
no I mean the actual name of this file - the file name is `UpscanRequest` and the case class is called `UploadRequest`

Reply:
lol - fixed